src/content/containers/TimelineView.js

```diff
@@ -53,26 +62,35 @@ class TimlineViewTimelinesImpl extends PureComponent {
                        order={threadOrder}
                        orient='vertical'
                        onChangeOrder={changeThreadOrder}>
-            {threads.map((thread, threadIndex) => (
-              <div className='timelineViewRow' key={threadIndex}>
-                <TimelineFlameChart threadIndex={threadIndex}
-                                    viewHeight={height}
-                                    getScrollElement={this.getScrollElement} />
-              </div>
+            {threads.map((_, threadIndex) => (
+              hiddenThreads.includes(threadIndex)
+                ? <div className='timelineViewRowHidden' />
```

Comment:
I chose to simplify the logic here for moving threads, and used the built-in logic for `Reorderable`. Hidden threads are still rendered as actual divs, but they have a height of 0 and no pointer events sent to them. I followed this practice on all `Reorderable` instances.